pkg/bitcoin/htlc/database/server.ts

```diff
@@ -0,0 +1,31 @@
+import express from "express"
+import orderCollection from "./model/dbSchema"
+import "./connection"
```

Comment:
don't use this hacky way, please use a proper index.ts for setup

Reply:
connection is our index file

Reply:
it should be named "index.ts" though to disable coverage and have a uniform entrypoint for our docker-compose file